src/store/helium/heliumDataSlice.ts

```diff
@@ -66,12 +65,7 @@ export const fetchInitialData = createAsyncThunk<HeliumDataState>(
 const heliumDataSlice = createSlice({
   name: 'heliumData',
   initialState,
-  reducers: {
-    signOut: () => {
-      signOut()
-      return { ...initialState, isRestored: true }
```

Comment:
So we save this data because its not related to an account?

Reply:
yeah, there's no reason to reset it. It's all account agnostic.